mean_test.hhs

```diff
@@ -0,0 +1,21 @@
+/**
+ * @author Jianan Lin (林家南)
+ */
+
+*import math: mean
+
+let a = 1;
+let b = [5,34,3];
+let c = [[3,2,3],[3,4,2]];
+let d = new Mat([3,4,2]);
+let e = new Mat([[3,4],[4,3],[4,3]]);
+let f = [[[2,2],[3,3],[4,4]],[[2,2],[3,3],[4,4]],[[4,4],[3,3],[2,2]]];
+let g = [[1, 2], [3], [4, 5, 6]];
+
+print(mean(1, 2));
+print(mean(b));
```

Comment:
or b, we look up on wolfram, or calculator (5+34+3)/3 = 42/3 = 14 so we expect 14.
so instead of `print(mean(b))`
do,
```js
if ( !(mean(b) === 14 ) {
    throw new Error('Unit test case failed for b = [5,34,3]');
}
```
and so on.